lib/ansible/modules/copy.py

```diff
@@ -79,9 +79,10 @@
       See CVE-2020-1736 for further details.
   directory_mode:
     description:
-    - When doing a recursive copy set the mode for the directories.
-    - If this is not set we will use the system defaults.
-    - The mode is only set on directories which are newly created, and will not affect those that already existed.
+    - Set the access permissions of newly created directories to the given mode.
+      Permissions on existing directories do not change.
+    - See C(mode) for the syntax of accepted values.
```

Comment:
This should have been `O(mode)` since `mode` is a module option. I've created a PR (#81590) to update the markup.